oauthproxy.go

```diff
@@ -759,27 +759,27 @@ func (p *OAuthProxy) OAuthCallback(rw http.ResponseWriter, req *http.Request) {
 	// finish the oauth cycle
 	err := req.ParseForm()
 	if err != nil {
-		logger.Printf("Error while parsing OAuth2 callback: %v", err)
+		logger.Errorf("Error while parsing OAuth2 callback: %s" + err.Error())
```

Comment:
I think it's better if we keep them all as `%v` and `err`, this one was concatenating for some reason though?
```suggestion
		logger.Errorf("Error while parsing OAuth2 callback: %v", err)
```

Reply:
That makes no sense.
Very confused where these changes came from. I'm either losing my mind or this wasn't there before this morning's rebase.